memdx/ops_crud.go

```diff
@@ -155,6 +155,13 @@ func (o OpsCrud) decodeCommonError(resp *Packet) error {
 	return OpsCore{}.decodeError(resp)
 }
 
+func (o OpsCrud) wrapErrorWithServerDuration(err error, serverDuration time.Duration) *ErrorWithServerDuration {
+	return &ErrorWithServerDuration{
```

Comment:
I think this will end up wrapping errors with a ErrorWithServerDuration object even in cases where the server does not support server durations, or did not pass one back, is that going to be an issue?  I wonder if the server returning "0" is valid, if not, you can probably just do something like..
```
if serverDuration == 0 {
return err
}